refinery/ui/source/js/data-set-import/services/import-confirmation-service.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
```

Comment:
should be wrapped in an iffy and https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y040 

Angular Tool module should be up to date with code style also.